code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -23,9 +23,14 @@
 		return
 	A.attack_ghost(src)
 	if(ishuman(A) && in_range(src, A))
-		Harvest(A)
+		if(isLivingSSD(A))
+			if(client && client.send_ssd_warning(A)) //Do NOT Harvest(SSD people)
+				return
+			else
+				Harvest(A)
```

Comment:
I think you might want to check your logic here. What you have now will *only* harvest if the user is SSD and the ssd warning gets sent.

Reply:
Fixed in 5ad3a2d